bn-expansions/package.json

```diff
@@ -1,27 +1,27 @@
 {
-    "name": "@boatnet/bn-expansions",
-    "version": "0.0.1",
-    "main": "lib/index.js",
-    "types": "lib",
-    "scripts": {
-      "build": "tsc -p ."
-    },
-    "author": "FRAM Data Team",
-    "license": "ISC",
-    "repository": {
-      "type": "git",
-      "url": "https://github.com/nwfsc-fram/boatnet-module.git"
-    },
-    "dependencies": {
-      "geojson": "^0.5.0",
-      "davenport": "^2.8.2",
```

Comment:
might consider removing davenport if you're not using it.

Reply:
okay removed it